### PR TITLE
edited the proposal

### PR DESCRIPTION
**Pull request by Shekharrajak**

### CODE REVIEW

**Comment by Shekharrajak** [comment]
Ping @ranjithkumar007 @ShikharJ see the changes. I am creating the draft.

File: README.md [hunk 1]
```diff
@@ -31,13 +36,19 @@ SymPy India developers will be conducting the workshop:
...
 
-SymPy is a pure Python library for symbolic mathematics. It aims to become a full-featured computer algebra system (CAS) while keeping the code as simple as possible in order to be comprehensible and easily extensible. SymPy is written entirely in Python and does not require any external libraries. The tutorial will cover the following topics and more.
+We expect attendees of this tutorial to have basic knowledge of Python, C++, Ruby, Julia and mathematics. However, many more advanced topics for SymPy will be explained during presentation and basic examples for SymEngine wrappers. No prior knowledge of SymPy or other Python libraries is required, although it is suggested that attendees be familiar with the IPython notebook
```
Comment (ShikharJ) [inline README.md:27]: A full stop at the end, though I'm really stretching it.

**Comment by Shekharrajak** [comment]
The proposal link is here :

 https://in.pycon.org/cfp/2017/proposals/sympy-symengine-and-its-wrapper-for-different-programming-languages-as-computer-aided-algebra-system-cas~eXnob/

Please have a look and let me know if you find any mistake or want to add something.

**Comment by Shekharrajak** [comment]
Feel free to edit the proposal here : https://github.com/sympy/sympy/wiki/PyCon-India-2017-Proposal and let me know. @ShikharJ @ranjithkumar007

**Pull request merged**